pages/vi/profiles/ArcyFlores.md

```diff
@@ -0,0 +1,28 @@
+# Hello! 
+My name is **Arcy Flores**
+ 
+ ## Details
+  * Current Location: Boston, MA
+  * Eastern Standard Time
+  * mac os High Sierra
+  
+ ### About Me
+  I am a former Propety Management Professional now transitioning into Computer Science. I've managed 
+  to transition by attending the [ALIGN Program](https://align.ccis.northeastern.edu/) at Northeastern University.
+  
+ ### Languages/Tools/Software
+ * Java
+ * Javascript
+ * HTML/CSS
+ * Node.js
+ * Elixir
+ * Phoenix Framework
+ * IntelliJ
+ 
+ ### Links
+ [Github](https://github.com/ArcyFlores)
+ [LinkedIn](www.linkedin.com/in/arcy-flores)
+ 
+ 
+ 
+  
```

Comment:
Please also watch out for extra line breaks and spaces in general.

Reply:
Ok thank you! Will update accordingly